canonical-websites/www.ubuntu.com
# Updated /legal/ubuntu-advantage-service-terms

## Done

- Updated the short form service description and converted it to markdown

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-terms](http://0.0.0.0:8001/legal/ubuntu-advantage-service-terms)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1W0WJrXaIvC54MDPgS62xfgPhqJaSE7RcZ2mjIqz1Y2o/edit)


## Issue / Card

Fixes #4667
